src/doc/dev_manual/test_suite.rst

```diff
@@ -44,45 +43,51 @@ Here is an example of the contents of the generated ``run_visit_test_suite.sh``
        -e /Users/harrison37/Work/github/visit-dav/visit/build-mb-develop-darwin-10.13-x86_64/build-debug/bin/visit "$@"
 
 
-Once the test suite has run, the results can be found in the ``output/html`` directory. Open ``output/html/index.html`` in a web browser to view the test suite results.
+Once the test suite has run, the results can be found in the ``output/html`` directory.
+Open ``output/html/index.html`` in a web browser to view the test suite results.
+
+If you want to restrict the amount of parallelism used in running the test suite you can do so with the ``-n`` command line option.
+By default, the test suite will be run using all the cores on your system.
+We have found that on some systems, running more than one test at a time may result in failures.
+To work around this issue you can run one test at a time. ::
+
+    ./run_visit_test_suite.sh -n 1
+
+If you want to run a single test or just a few tests from the test suite you can list them on the command line.
+The list of tests must be the last entries on the command line. ::
+
+    ./run_visit_test_suite.sh -n 1 tests/databases/silo.py tests/databases/xdmf.py
+
+There are a number of additional command-line options to the test suite.
+``./run_visit_test_suite.sh -help`` will give you details about these options.
 
 Accessing regression test results
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The nightly test suite results are posted to: http://portal.nersc.gov/project/visit/.
+The nightly test suite results are posted to `GitHub <https://visit-dav.github.io/dashboard/>`_.
 
 In the event of failure on the nightly run
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If any tests
-fail, ''all'' developers who updated the code from the last time all
-tests successfully passed will receive an email indicating what failed.
+If any tests fail, ''all'' developers who updated the code from the last time all tests successfully passed will receive an email indicating what failed.
```

Comment:
what do the double single quotes surrounding *all* do? Is that for bold? I thought reStructuredText required double asterisks for that.

Reply:
I suspect that the intent was to make it bold. I will change the single quotes to asterisks.